packages/components/src/menu-item/index.js

```diff
@@ -68,6 +69,7 @@ export function MenuItem( props, ref ) {
 				shortcut={ shortcut }
 			/>
 			{ icon && iconPosition === 'right' && <Icon icon={ icon } /> }
+			{ suffix }
```

Comment:
Since this is a new API and we can kind of "start from scratch", what do you think about **not** rendering the `shortcut` and the `right` icon element if a `suffix` is used? I say this because I feel like we might want to eventually deprecate the `icon` and `shortcut` props here and unify everything into the `prefix`/`suffix` props. That would make things simpler for this component itself, and also better for API consistency across the package.

Reply:
That sounds like a good plan to me 👍 

I've updated `MenuItem` according to your suggestion in https://github.com/WordPress/gutenberg/pull/44260/commits/90627621cdf3fd5b7e0b40e4bfe3a565fe97293d

Reply:
Looks good!